README.md

```diff
@@ -321,11 +321,22 @@ See [Quick Start](#quick-start) how to launch Houdini with Palladio.
 
 ## Release Notes
 
-### v2.0.0 Beta 2
+### v2.0.0 Beta 2 (Mar 27, 2023)
 
 Required CityEngine version: 2022.1 or older.
+
 #### Changed:
-- Updated Procedural Runtime (PRT) to 2.7.8538 (corresponds to CityEngine 2022.1)
+- Changed indexing of generated meshes: all vertex attributes now have separate indices. This improves mesh connectivity and sharing of points.
+- Fixed rule attribute overrides for multiple initial shapes.
+- Fixed two bugs in the "FileSystem Helper" when reading textures from RPKs (fixes rendering with Mantra). 
+- Updated Procedural Runtime (PRT) to 2.7.8538 (corresponds to CityEngine 2022.1).
```

Comment:
we did not use any "." in the other comments, but that's minor. Feel free to change it or leave it.